From 53b056ef7f7c0dfda981701f0236ca8704a787c9 Mon Sep 17 00:00:00 2001
From: Yihang Li <liyihang9@huawei.com>
Date: Tue, 16 Jun 2026 11:20:51 +0800
Subject: [PATCH 05231/15333] scsi: MAINTAINERS: Update HiSilicon hisi_sas
 driver maintainer to Xingui Yang

Replace myself with Xingui Yang who is very familiar with the HiSilicon
hisi_sas drivers.

Signed-off-by: Yihang Li <liyihang9@huawei.com>
Reviewed-by: Jason Yan <yanaijie@huawei.com>
Link: https://patch.msgid.link/20260616032051.1268608-1-liyihang9@huawei.com
Signed-off-by: Martin K. Petersen <martin.petersen@oracle.com>
---
 MAINTAINERS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/MAINTAINERS b/MAINTAINERS
index 15011f5752a9..9a66b11ea521 100644
--- a/MAINTAINERS
+++ b/MAINTAINERS
@@ -11788,7 +11788,7 @@ F:	Documentation/devicetree/bindings/infiniband/hisilicon-hns-roce.txt
 F:	drivers/infiniband/hw/hns/
 
 HISILICON SAS Controller
-M:	Yihang Li <liyihang9@h-partners.com>
+M:	Xingui Yang <yangxingui@huawei.com>
 S:	Supported
 W:	http://www.hisilicon.com
 F:	Documentation/devicetree/bindings/scsi/hisilicon-sas.txt
-- 
2.55.0

